src/delaunay.js

```diff
@@ -95,10 +93,11 @@ export default class Delaunay {
   }
   renderHull(context) {
     const buffer = context == null ? context = new Path : undefined;
-    const {hull} = this;
-    let node = hull;
-    context.moveTo(node.x, node.y);
-    while (node = node.next, node !== hull) context.lineTo(node.x, node.y);
+    const {hull, points} = this;
+    for (let i = 0, h; i < hull.length; ++i) {
+      h = hull[i];
+      context[i ? "lineTo" : "moveTo"](points[2 * h], points[2 * h + 1]);
+    }
```

Comment:
I’d prefer the first context.moveTo broken out of the loop (so you don’t need to test `i` inside the loop). Also pull out `n`, and then `const h = hull[i] * 2;` inside the loop.